src/config/environment.js

```diff
@@ -0,0 +1,5 @@
+const config = {
+  backendUrl: 'http://localhost:3000/api/v1'
```

Comment:
Why is this `http://localhost:3000/api/v1` but environment-dev is `http://localhost:3000`?